### DISCUSSION TITLE
Setup a Single-Node Cluster fails

### DISCUSSION THREAD

**Discussion by mahille** [vhive-serverless/vHive#884] [Q&A]
Hi,

I am trying to work my way through the quickstart guide. But neither of the setups worked so far for me. I guess the single-node setup is the easier one, so let's start with that.
When I execute the steps according to the readme it fails to complete the last step:

`./setup_tool create_one_node_cluster firecracker`

The log output is:

```
[16:01:53] [Info] Untainting master >>>>> [16:01:53] [Error] [exit 1] -> The connection to the server 10.0.2.15:6443 was refused - did you specify the right host or port?
[16:01:53] [Error] Failed to untaint master!
[16:01:53] [Error] Faild subcommand: create_one_node_cluster!
```

Where can I find further information why this fails?

Maybe some more info on my setup: I run this in a qemu-based VM with two network adapters, one is the qemu user-level networking (10.0.2.15/24) for internet access and the second is a tap device (192.168.123.10/24) which is connected to a bridge in order to connect to the VM from the host (and possibly from other VMs for a multi-node setup). Might this two-adapter setup be confusing for vHive?

**Comment by leokondrashov** [comment]
You might be right that it is networking problem, although, it shouldn't be confused about the two-adapter setup since the CloudLab nodes that we use also have two NICs attached. Possible problem might be with subnets.

Also problem might be with the usage of qemu. I'm not sure that vHive was tested for it.

Please provide following information about the setup process so we can help to resolve the issue:
* Was this the first error that you saw? If the k8s cluster failed to be created, it should fail before the step of untainting master, on the step that prints "Creating cluster".
* The content of files `/etc/kubernetes/admin.conf` and `~/.kube/config`.
* Also you can try to find whether is there something on 6443 port on any of the IPs, maybe indeed it detected the wrong IP during setup.
